saki8093/hw1b/variables.tf

```diff
@@ -0,0 +1,54 @@
+variable "region" {
+  default = "us-east-2"
+}
+variable "ami" {
+  default = "ami-965e6bf3"
+}
+variable "inst_type" {
+  description = "Instance type t2.micro"
+  default = "t2.micro"
+}
+
+variable "user" {
+	default="admin"
+}
+
+variable "ssh_keyfile" {
+  default = "/home/sampykishan/Desktop/AWSkeys/Terraform_2.pem"
+}
+
+
+variable "aws_access_key" {
+  default = "AKIAJHMMAAKXYWPSZBSA"
```

Comment:
If these are valid keys you'll want to go into your account ASAP and update/delete them, never put access keys in github public repos.  If those are modified/dummy keys that's no problem.